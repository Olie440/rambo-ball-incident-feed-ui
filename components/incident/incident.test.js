import { shallow } from 'enzyme';

import MockGame from '~/__mocks__/mock-game';
import Incident from './incident.component';

describe('<Incident />', () => {
    it('renders the correct HTML', () => {
        const props = MockGame().incidents[0];
        const element = shallow(<Incident {...props} />);

        expect(element).toMatchSnapshot();
    });
});