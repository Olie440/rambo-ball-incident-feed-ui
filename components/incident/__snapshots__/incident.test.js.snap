// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Incident /> renders the correct HTML 1`] = `
<div
  className="incident"
>
  <div>
    15:34:53
  </div>
  <div>
    GOAL_MISSED_AWAY
  </div>
</div>
`;
